do-not-commit/unchecked-returns.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
+import subprocess as sub
+
+# ok
+x = sub.call("foo")
+
+# ruleid: unchecked-subprocess-call
+sub.call("foo")
```

Comment:
This is not checking the return value of this subprocess call; if it fails no exception will be raised. Consider subprocess.check_call() instead ([.unchecked-returns.unchecked-subprocess-call](https://semgrep.dev/r/.unchecked-returns.unchecked-subprocess-call))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=.unchecked-returns.unchecked-subprocess-call) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>